modules/network-observability-RTT.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-RTT_{context}"]
+= Working with RTT tracing
+You can track RTT by editing the `FlowCollector` to the specifications in the following YAML example.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. In the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*.
+. Select *cluster*, and then select the *YAML* tab.
+. Configure the `FlowCollector` custom resource for RTT tracing, for example:
++
+[id="network-observability-flowcollector-configuring-RTT_{context}"]
+.Example `FlowCollector` configuration
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: Direct
+  agent:
+    type: eBPF
+    ebpf:
+      features:
+       - FlowRTT               <1>
+      sampling: 1              <2>
+----
+<1> You can start tracing RTT network flows by listing the `FlowRTT` parameter in the `spec.agent.ebpf.features` specification list.
+<2> Optionally, consider setting `sampling` to a value of `1` for more accurate metrics.
```

Comment:
@msherif1234 Think we need to remove this statement regarding sampling right?

Reply:
yes